app/models/log_spreadsheet.rb

```diff
@@ -80,12 +80,7 @@ def create_spreadsheet(columns, logs)
       end
       row = sheet.row(row_idx)
       columns.each do |col|
-        if TIME_COLS.include?(col)
-          val = log[col].to_datetime
-        else
-          val = log.value(col)
-        end
-        row.push(val)
+        row.push(log.value(col))
```

Comment:
Other casual data reviewers will like to see the actual date string. Since we are using CSV now, using the timestamp seems best.  But in the case of XLS it would be better to store the datetime object, (assuming it shows up as a date field in XLS). And we need to give Paul a lesson in Excel macros dealing with date fields. :)